dsl_parser/elements/imports.py

```diff
@@ -253,7 +258,10 @@ def _merge_into_dict_or_throw_on_duplicate(from_dict_holder, to_dict_holder,
                    "on '{1}'".format(key_name, key_holder.value))
 
 
-def _fetch_import(import_url):
+def _fetch_import(import_url, resolver):
+    url_parts = import_url.split(':')
+    if url_parts[0] in ['http', 'https', 'ftp']:
+        return resolver.resolve(import_url)
```

Comment:
consider wrapping the possible implementation raised exception which can be of any type with DSLParsingResolverException using try/catch.